src/Orleans/CodeGeneration/GrainInterfaceUtils.cs

```diff
@@ -126,6 +126,9 @@ public static bool IsStatelessWorker(MethodInfo methodInfo)
         
         public static int ComputeMethodId(MethodInfo methodInfo)
         {
+            var attr = methodInfo.GetCustomAttribute<MethodIdAttribute>(true);
+            if (attr != null) return attr.MethodId;
```

Comment:
What will happen if attribute with same MethodId will be applied to multiple methods of interface?